assets/src/edit-story/app/layout/layout.js

```diff
@@ -72,11 +74,13 @@ const Area = styled.div`
 function Layout() {
   return (
     <LayoutProvider>
-      <Editor>
-        <Area area="lib">
-          <Library />
-        </Area>
-        <Workspace />
+      <Editor zIndex={3}>
+        <CanvasProvider>
```

Comment:
Since you add `CanvasProvider` here, need to remove it from its prior location in `Workspace` (`workspace.js`)?

Reply:
It is removed there as far as I can see: https://github.com/google/web-stories-wp/pull/5512/files#diff-cc8f92173d9a285323c8490a7626429e4f8270ab26428d73eac9e52048401f0dR27